### PR TITLE
Remove newlines between uncommented interface method signatures

### PR DESCRIPTION
**Pull request by Oiyoo**
this addresses the comments on PR#95 since the author ghosted

implements the suggestion for interfaces discussed in #85

### CODE REVIEW

File: format/format.go [hunk 1]
```diff
@@ -436,6 +436,26 @@ func (f *fumpter) applyPre(c *astutil.Cursor) {
 			node.Rparen = token.NoPos
 		}
 
+	case *ast.InterfaceType:
+		methods := node.Methods.List
+		for i := range methods {
+			curMethodPos := methods[i].Pos()
```
Comment (mvdan) [inline format/format.go:442]: how about:

```
var prev *ast.Field
for _, method := range node.Methods.List {
    if prev == nil { ... }
    ...
    prev = method
}
```

File: format/format.go [hunk 2]
```diff
@@ -436,6 +436,26 @@ func (f *fumpter) applyPre(c *astutil.Cursor) {
...
+			if i == 0 {
+				if comments := f.commentsBetween(node.Interface, curMethodPos); len(comments) > 0 {
+					// only rm leading line upto the first comment
+					curMethodPos = comments[0].Pos()
+				}
+				// rm leading lines if they exist
```
Comment (mvdan) [inline format/format.go:448]: nit: please spell things out in the comments, like "remove" instead of "rm" :)

**Review by mvdan — APPROVED** [review]
Very nice :) just two nits.

File: format/format.go (hunk 1 above)
Comment (Oiyoo) [reply format/format.go]: thanks! PTAL

**Pull request merged**